### PR TITLE
Update Book for 0.5

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Updated for 0.5! Please review to try and catch issues before our new users do :p

Style-wise, I'm not sure I prefer `commands.spawn_batch()` over `commands.insert().insert()` for the initial Hello World example. Let me know your preferences and we can change it.

### CODE REVIEW

File: content/learn/book/getting-started/ecs/_index.md [hunk 1]
```diff
@@ -78,11 +78,11 @@ struct Name(String);
...
 ```rs
-fn add_people(commands: &mut Commands) {
+fn add_people(mut commands: Commands) {
     commands
-        .spawn((Person, Name("Elaina Proctor".to_string())))
-        .spawn((Person, Name("Renzo Hume".to_string())))
-        .spawn((Person, Name("Zayna Nieves".to_string())));
+        .spawn_bundle((Person, Name("Elaina Proctor".to_string())))
+        .spawn_bundle((Person, Name("Renzo Hume".to_string())))
+        .spawn_bundle((Person, Name("Zayna Nieves".to_string())));
```
Comment (jakobhellermann) [inline content/learn/book/getting-started/ecs/_index.md]:
```suggestion
    commands.spawn_bundle((Person, Name("Elaina Proctor".to_string())));
    commands.spawn_bundle((Person, Name("Renzo Hume".to_string())));
    commands.spawn_bundle((Person, Name("Zayna Nieves".to_string())));
```

File: content/learn/book/getting-started/_index.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Add the bevy crate to your project's Cargo.toml like this:
 
 ```toml
 [dependencies]
-bevy = "0.4" # make sure this is the latest version
+bevy = "0.5" # make sure this is the latest version
```
Comment (jakobhellermann) [inline content/learn/book/getting-started/_index.md:41]: The [setup page](https://bevyengine.org/learn/book/getting-started/setup/) has a reference to bevy 0.4 as well, I think you need to update there as well.

**Review by jakobhellermann — CHANGES_REQUESTED** [review]

File: content/learn/book/getting-started/ecs/_index.md (hunk 1 above)
Comment (alice-i-cecile) [reply content/learn/book/getting-started/ecs/_index.md]: Politely disagree. I don't mind the style, but this is inconsistent with how commands are used in all of the other official examples.

Comment (jakobhellermann) [reply content/learn/book/getting-started/ecs/_index.md]: My point was that the `commands.` in the beginning were missing, not the spawn_bundle vs insert.

**Comment by cart** [comment]
Looks good to me! I also think starting without Bundles is a good idea.

**Comment by jakobhellermann** [comment]
I made the suggestion while `spawn_bundle` was still used.

What I actually meant is that without the `commands.` at the beginning of each line the example fails to compile, since the chaining doesn't work anymore.

**Comment by jakobhellermann** [comment]
I opened a follow up PR to fix the compile error: #116

**Pull request merged**